meta-openstack/recipes-support/chef/chef_git.bb

```diff
@@ -11,14 +11,14 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=8f7bb094c7232b058c7e9f2e431f389c"
 
 PR = "r0"
 
-BPV = "12.4.1"
+BPV = "12.5.1"
```

Comment:
Would you mind breaking out the change to upgrade `chef` to `12.5.1` into its own commit - just so that we can track it separately?

Reply:
I tried that, but the tag is missing from the new location. So I can make it separate, but it won't be in a buildable state in between. If you're ok with that, then I'll push an update.

Reply:
Ah. That's a reasonable justification.